data/json/recipes/weapon/ranged.json

```diff
@@ -1073,6 +1073,38 @@
     "decomp_learn": 0,
     "components": [ [ [ "m320_mod", 1 ], [ "m320_mod_mod", 1 ] ], [ [ "ugl_buttstock", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "m26_mass_standalone",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_RANGED",
+    "skill_used": "shotgun",
+    "time": "5 m",
+    "reversible": true,
+    "autolearn": true,
+    "decomp_learn": 0,
+    "components": [
+      [ [ "m26_mass", 1 ], [ "m26_mass_mod", 1 ] ],
+      [
+        [ "pistol_grip", 1 ],
+        [ "adjustable_stock", 1 ],
+        [ "folding_stock", 1 ],
+        [ "folding_stock_folded", 1 ],
+        [ "high_end_folding_stock", 1 ],
+        [ "high_end_folding_stock_folded", 1 ],
+        [ "wire_stock", 1 ],
+        [ "wire_stock_folded", 1 ],
+        [ "pistol_stock", 1 ],
+        [ "recoil_stock", 1 ],
+        [ "cheek_pad", 1 ],
+        [ "butt_hook", 1 ],
+        [ "tail_hook_stock", 1 ],
+        [ "under_folding_stock", 1 ],
+        [ "under_folding_stock_folded", 1 ]
```

Comment:
Having all these options that become a regular stock once the item is crafted is a logic hole :P